sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -491,11 +491,6 @@
             This rule is used to enforce that all modules depend on the same version of particular dependencies or
             plugins.
             -->
-            <requireSameVersions>
-              <dependencies>
-                <dependency>com.azure.*</dependency>
```

Comment:
Should this be changed to `com.azure:*` instead of being removed?

Reply:
I don't know what you really want to do. As I tested, if I add `com.azure:*`, all data-plane SDK will build with failure, since all their versions are different

Reply:
like: https://github.com/Azure/azure-sdk-for-java/blob/10bf934/sdk/storage/azure-storage-blob/pom.xml#L55-L69